gatsby-config.js

```diff
@@ -57,9 +60,10 @@ module.exports = {
               },
             },
           },
-          {
-            resolve: `${__dirname}/src/gatsby-remark-social-card`,
-          },
+          
+          // {
+          //   resolve: `${__dirname}/src/gatsby-remark-social-card`,
+          // },
```

Comment:
Do we still need this or can it be removed?

Reply:
My error, we still need this